srcpkgs/libjxl/template

```diff
@@ -0,0 +1,78 @@
+# Template file for 'libjxl'
+pkgname=libjxl
+version=0.6.1
+revision=1
+#_short_version=0.6
+#wrksrc=libjxl-${_short_version}
+build_style=cmake
+configure_args="-DJPEGXL_ENABLE_BENCHMARK=OFF -DJPEGXL_ENABLE_EXAMPLES=OFF
+ -DJPEGXL_ENABLE_SJPEG=OFF -DJPEGXL_ENABLE_PLUGINS=ON"
+hostmakedepends="tar pkg-config asciidoc"
+makedepends="brotli-devel highway libpng-devel giflib-devel libjpeg-turbo-devel
+ libopenexr-devel gdk-pixbuf-devel gimp-devel"
+checkdepends="gtest-devel xdg-utils"
+short_desc="JPEG XL image format reference implementation"
+maintainer="Joshua Krämer <joshua@kraemer.link>"
+license="BSD-3-Clause, custom:Patent grant"
+homepage="https://jpeg.org/jpegxl/"
+_lodepng_hash=48e5364ef48ec2408f44c727657ac1b6703185f8
+_skcms_hash=64374756e03700d649f897dbd98c95e78c30c7da
+distfiles="https://github.com/libjxl/libjxl/archive/v${version}.tar.gz
+ https://github.com/lvandeve/lodepng/archive/${_lodepng_hash}.tar.gz>lodepng-${_lodepng_hash}.tar.gz
+ https://skia.googlesource.com/skcms/+archive/${_skcms_hash}.tar.gz>skcms-${_skcms_hash}.tar.gz"
+checksum="ccbd5a729d730152303be399f033b905e608309d5802d77a61a95faa092592c5
+ c47c48c77a205f1af484b7b5a847290af65de3ea6f15817aa27c5ec7cc5208fd
+ @a69230c7b6f03a178c93abc6edc832c040d6e198340193b27c3d04afecf8f617"
+skip_extraction="lodepng-${_lodepng_hash}.tar.gz
+ skcms-${_skcms_hash}.tar.gz"
```

Comment:
Could we instead use create_wrksrc and build_wrksrc?

Reply:
It would be more complicated because the third-party packages would still need to be moved to specific subdirectories after extraction.